tests/suites/test_suite_psa_crypto.function

```diff
@@ -9479,6 +9481,33 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void ecc_conversion_functions(int grp_id_arg)
+{
+    mbedtls_ecp_group_id grp_id = grp_id_arg;
+    psa_ecc_family_t ecc_family;
+    size_t bits;
+
+    ecc_family = mbedtls_ecc_group_to_psa(grp_id, &bits);
+    TEST_ASSERT(ecc_family != 0);
+    TEST_EQUAL(grp_id, mbedtls_ecc_group_from_psa(ecc_family, bits));
```

Comment:
This is only testing that the functions are inverse of each other, not that they are correct. The risk of errors here seems low, but as a matter of principle I'd like it better if the test functions took parameters like `exp_ecc_family` and `exp_bits` and checked that family and bits are as expected.